x-pack/plugins/security_solution/public/detections/components/take_action_dropdown/index.test.tsx

```diff
@@ -240,6 +240,13 @@ describe('take action dropdown', () => {
         ).toEqual('Respond');
       });
     });
+    test('should render "Apply alert tags"', async () => {
+      await waitFor(() => {
+        expect(
+          wrapper.find('[data-test-subj="alert-tags-context-menu-item"]').first().text()
+        ).toEqual('Apply alert tags');
```

Comment:
I think you can also just use the i18n string so then in the future if you test this, you don't have to update here, just makes sure it's pulling the expected string.